usecase/src/test/java/org/mobilitydata/gtfsvalidator/usecase/ValidateRouteColorAndTextContrastTest.java

```diff
@@ -130,7 +130,7 @@ void sameRouteColorAndTextShouldGenerateNotice() {
         verify(mockRoute, times(2)).getRouteColor();
         verify(mockRoute, times(2)).getRouteTextColor();
         verify(mockRoute, times(1)).getRouteId();
-        verify(mockResultRepo, times(1)).addNotice(any(ErrorNotice.class));
+        verify(mockResultRepo, times(1)).addNotice(any(RouteColorAndTextInsufficientContrastNotice.class));
```

Comment:
Here, adding a line return would be useful to avoid exceeding the maximum number of character on a single line